src/components/elements/carousel/carousel.js

```diff
@@ -36,6 +38,16 @@ PrevButton.propTypes = {
     onClick: PropTypes.func,
 }
 
+export const NavigationButton = ({ color, is_enabled, onClick }) => (
+    <StyledDot onClick={onClick} color={is_enabled ? color : null} />
```

Comment:
```suggestion
    <StyledDot onClick={onClick} color={is_enabled && color} />
```

Reply:
`is_enabled && color` will be evaluated to true. In the logic of `StyleDot` component the value needs to be a string describing the color